clickhouse_sqlalchemy/__init__.py

```diff
@@ -3,13 +3,10 @@
 from .orm.session import make_session
 from .sql import Table, MaterializedView, select
 
-
-VERSION = (0, 3, 2)
-__version__ = '.'.join(str(x) for x in VERSION)
```

Comment:
Honestly, I have not found an easier and cheaper way for uv to use dynamic versioning.

This solution - [uv-dynamic-versioning](https://pypi.org/project/uv-dynamic-versioning/) - aims to retrieve data from a VCS.

I tried this solution - [setuptools-scm](https://pypi.org/project/setuptools-scm/) - yet it didn't work (an unexpected `version_file` param or smth).

So, if you can find a better solution which allows the uv manager to support dynamic versioning in a cheap and an easy way, feel free to coautor and propose changes and even commits.